app/src/forms/common/constants.js

```diff
@@ -29,7 +29,7 @@ module.exports = Object.freeze({
     FORM_API_UPDATE: 'form_api_update',
     FORM_DELETE: 'form_delete',
     FORM_READ: 'form_read',
-    FORM_SUBMITTER: ['form_read', 'submission_create'],
+    FORM_SUBMITTER: ['form_read', 'submission_create', 'document_template_read'],
```

Comment:
This permission is used for the idp users.

Reply:
Just an FYI: This new `FORM_SUBMITTER` will be removed in the future: it is not a permission, it's a role. It's going to be a bit of work to figure out how to best provide these permissions for that role.
https://bcdevex.atlassian.net/browse/FORMS-1452

Reply:
Yes, I noticed that, and it makes it confusing. It's used to apply a few permissions together in one spot. It should be a speedy fix I think.